Documentation/gettingstarted/hubble-configuration.rst

```diff
@@ -24,212 +24,272 @@ This allows Hubble Relay to communicate with all Hubble instances in the
 cluster. Connections between Hubble instances and Hubble Relay are secured using
 mutual TLS (mTLS) by default.
 
-TLS certificates can be provided by manually on the Helm install command (user
-provided) or generate automatically via either:
+TLS certificates can be generate automatically or provided by manually.
+
+The following options are available to configure TLS certificates automatically:
 
-* `Helm <https://helm.sh/docs/chart_template_guide/function_list/#gensignedcert>`__
-* cilium's `certgen <https://github.com/cilium/certgen>`__ (using a Kubernetes ``CronJob``)
 * `cert-manager <https://cert-manager.io/>`__
+* cilium's `certgen <https://github.com/cilium/certgen>`__ (using a Kubernetes ``CronJob``)
+* `Helm <https://helm.sh/docs/chart_template_guide/function_list/#gensignedcert>`__
 
-Auto generated certificates via Helm
-------------------------------------
+.. tabs::
 
-When using Helm, TLS certificates are (re-)generated every time Helm is used
-for install or upgrade. As Hubble server and Hubble Relay support TLS
-certificates hot reloading, including CA certificates, this does not disrupt
-any existing connection. New connections are automatically established using
-the new certificates without having to restart Hubble server or Hubble
-Relay.
+    .. group-tab:: cert-manager
 
-::
+        This method relies on `cert-manager <https://cert-manager.io/>`__ to generate
+        the TLS certificates. cert-manager has becomes the de facto way to manage TLS on
+        Kubernetes, and it has the following advantages compared to the other
+        documented methods:
```

Comment:
Should we mention that it's a bit more work to get working vs certgen as it requires installing and managing cert-manager in the first place? It's probably straightforward for users who already use cert-manager but may be slightly overwhelming for someone who is just enabling Hubble and wants TLS.

Reply:
I don't think it's really necessary to tell users that. I guess we can swap the order, between certgen and cert-manager and that should be sufficient. I think it's obvious enough that it has more steps than cert-gen.